satellite/gracefulexit/endpoint_test.go

```diff
@@ -1207,6 +1207,191 @@ func TestFailureNotFoundPieceHashUnverified(t *testing.T) {
 
 }
 
+func TestFailureStorageNodeIgnoresTransferMessages(t *testing.T) {
+	var maxOrderLimitSendCount int
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount:   1,
+		StorageNodeCount: 5,
+		UplinkCount:      1,
+		Reconfigure: testplanet.Reconfigure{
+			Satellite: func(logger *zap.Logger, index int, config *satellite.Config) {
+				// We don't care whether a node gracefully exits or not in this test,
+				// so we set the max failures percentage extra high.
+				config.GracefulExit.OverallMaxFailuresPercentage = 101
+				maxOrderLimitSendCount = config.GracefulExit.MaxOrderLimitSendCount
```

Comment:
I think we should verify that `maxOrderLimitSendCount` is > 1 or something. Either that, or manually configure it ourselves here.

Reply:
ok, set it to 3 now